infra-envs.yml

```diff
@@ -0,0 +1,15 @@
+- name: Test infra-envs
+  hosts: localhost
+  tasks:
+
+    - name: Create a infra-env
+      infra-envs:
```

Comment:
Use underscores instead of hyphens. Per https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html ...

"Module name MUST use underscores instead of hyphens or spaces as a word separator. Using hyphens and spaces will prevent ansible-core from importing your module."

Reply:
@clwheel Thanks for the comment and  sharing the link

Reply:
FYI I learnt something new here from this feedback

Reply:
Module name needs to be updated to `infra_envs` and playbook should be renamed `infra_envs.yml` for consistency.